documentation.md

```diff
@@ -22,7 +22,7 @@ const bodySegmentation = ml5.bodySegmentation(?modelName, ?options, ?callback);
 
 **Parameters:**
 
-- **modelName**: OPTIONAL: A string specifying which model to use, "SelfieSegmentation" or "BodyPix".
```

Comment:
I don't think this needs to be changed since the `modelName` parameter can still be set to either `SelfieSegmentation` or `BodyPix`. 

Here is the current `handleModelName` function for `bodySegmentation`:
```js
this.modelName = handleModelName(
      this.modelName,
      ["BodyPix", "SelfieSegmentation"], //<--- possible values for modelName
      "SelfieSegmentation", //<-- default value of modelName
      "bodySegmentation" //<-- name of the ml5 model, used by the friendly error system
);
```

Reply:
Thanks @ziyuan-linn - fixed.